hugegraph-llm/src/hugegraph_llm/operators/index_op/semantic_id_query.py

```diff
@@ -86,7 +86,7 @@ def run(self, context: Dict[str, Any]) -> Dict[str, Any]:
         graph_query_list = set()
```

Comment:
To keep single-text and batch paths consistent at query time, consider routing single-text calls through the same util layer (retry/limits/metrics) used by batch building. This reduces behavior drift between build and query.